site/profile/manifests/foreman_aio.pp

```diff
@@ -60,6 +60,15 @@
     },
   }
 
+  class { 'hiera':
+    hierarchy      => [
+      'nodes/%{::trusted.certname}',
```

Comment:
Yeah, I had to think about it a little too, but the array of values in the hierarchy parameter represent what should literally be in the `hiera.yaml` file. I basically just pulled from the main [example](https://forge.puppet.com/puppet/hiera/readme#beginning-with-hiera) though and modified it for this environment.